content/data-reporting/reports-api/v3.3/index.mdx

```diff
@@ -56,16 +56,17 @@ Shows the average chatting duration of agents within a license.
-| `tags`                | no       | Names of tags separated by a comma                                                                       |
-| `customer_client_ids` | no       | Client IDs separated by a comma                                                                          |
-| `distribution`        | no       | Possible values: `hour`, `day-hours`, `day`, `month`; defaults to `day`                                  |
-| `tagged`              | no       | Possible values: `true`, `1`, `false`, `0`                                                               |
+| Query String          | Required | Notes                                                                                                                                                                                                     |
+| --------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `to`                  | yes      | Date in the RFC3339 format. If `timezone` provided, the Timezone from this format will be ignored.                                                                                                        |
+| `from`                | yes      | Date in the RFC3339 format. If `timezone` provided, the Timezone from this format will be ignored.                                                                                                        |
+| `agents`              | no       | Agent emails separated by a comma; if not specified, returns the data for all agents within the license.                                                                                                  |
+| `groups`              | no       | Group IDs separated by a comma                                                                                                                                                                            |
+| `tags`                | no       | Names of tags separated by a comma                                                                                                                                                                        |
+| `customer_client_ids` | no       | Client IDs separated by a comma                                                                                                                                                                           |
+| `distribution`        | no       | Possible values: `hour`, `day-hours`, `day`, `month`; defaults to `day`                                                                                                                                   |
+| `timezone`            | no       | Timezone in the [TZ format](http://en.wikipedia.org/wiki/List_of_tz_database_time_zones) (e.g. America/Phoenix). Defaults to requester Timezone. If not present, `from` is parsed to get it's Timezone.   |
```

Comment:
> If not present, `from` is parsed to get **its** timezone.  

"its" suggest the requester is not a person, and I believe it's okay if we assume an application is the requester. But to avoid confusion, I suggest replacing "its" with the phrase "the requester's timezone".